.circleci/config.yml

```diff
@@ -42,10 +61,9 @@ jobs:
             black --check --diff test/
             isort --check --diff test/
 
-  build-test:
-    machine:
-      image: ubuntu-2204:2022.10.2
-    resource_class: 2xlarge
+  build:
+    # TODO this job could be run in Docker
```

Comment:
leaving this for now?

Reply:
Yeah, I've had enough of working on the CI so did plan on leaving this for now.

There are a few differences between the docker and machine Ubuntu images that'll take a little bit of time to work through.
Also, since we're building oid as a dynamically linked executable, we'd need to be certain we have the same shared libraries on the build and test jobs.